src/main/java/no/unit/nva/cristin/projects/RequestUtils.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.cristin.projects;
+
+import static nva.commons.core.attempt.Try.attempt;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import nva.commons.apigateway.RequestInfo;
+
+public class RequestUtils {
+
+    protected static final String LANGUAGE_QUERY_PARAMETER = "language";
+    protected static final String LANGUAGE_INVALID_ERROR_MESSAGE = "Parameter 'language' has invalid value";
+    protected static final String DEFAULT_LANGUAGE_CODE = "nb";
+    private static final List<String> VALID_LANGUAGE_CODES = Arrays.asList("nb", "en");
```

Comment:
VALID_LANGUAGE_CODES is a set, not a list. Use Set.of("en", "nb");

Reply:
In general, when order and presence of duplicates is unimportant use HashSet. It also has better performance for contains operations.